exonum/src/events/internal.rs

```diff
@@ -71,6 +76,23 @@ impl InternalPart {
                         }).map_err(|_| panic!("Can't execute shutdown"));
                         to_box(f)
                     }
+                    InternalRequest::VerifyTx(tx) => {
+                        let internal_tx = internal_tx.clone();
+                        let thread_pool = Rc::clone(&thread_pool);
+                        let f = future::lazy(move || {
+                            thread_pool.spawn(future::lazy(move || {
```

Comment:
Are we OK with a possible panic here? It seems that using `thread_pool.sender().spawn(_)`could be safer.